certified-connectors/DocuSignDemo/script.csx

```diff
@@ -1662,6 +1690,34 @@ public class Script : ScriptBase
       body["workflowIds"] = workflowsArray;
       response.Content = new StringContent(body.ToString(), Encoding.UTF8, "application/json");
     }
+  
+    if ("GetEnvelopeDocumentTabs".Equals(this.Context.OperationId, StringComparison.OrdinalIgnoreCase))
+    {
+      var body = ParseContentAsJObject(await response.Content.ReadAsStringAsync().ConfigureAwait(false), false);
+      var newBody = new JObject();
+      JArray tabs = new JArray();
+
+      foreach(JProperty tabTypes in body.Properties())
```

Comment:
You need to add a validation here to check if the document has tabs on it before looping through the body.Properties.

Reply:
if there are no tabs, it will just skip the foreach loop and return empty array, which is expected.